agent/php_error.c

```diff
@@ -407,14 +412,13 @@ static zval* nr_php_error_exception_stack_trace(zval* exception TSRMLS_DC) {
  *           will need to free, or NULL if no file name is available.
  */
 static char* nr_php_error_exception_file(zval* exception TSRMLS_DC) {
-  zval* file_zv = nr_php_call(exception, "getFile");
+  zval* file_zv = nr_php_get_zval_object_property(exception, "file");
   char* file = NULL;
 
   if (nr_php_is_zval_valid_string(file_zv)) {
     file = nr_strndup(Z_STRVAL_P(file_zv), Z_STRLEN_P(file_zv));
   }
 
-  nr_php_zval_free(&file_zv);
```

Comment:
When we obtained these values through nr_php_call, a copy was made that we needed to free. With directly calling get_*_property, there is no copy